docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ services:
     command: /code/django-config.sh
     volumes:
       - ./app/public:/code/public
-      - static_volume:/code/static
+      - django_static_volume:/code/static
```

Comment:
I don't understand why this is `/code/static` here but `/code/static/django` on line 52.

Reply:
Basically, I need the static files managed by django to be in a different directory than the static files managed in the nginx container. The static files managed by django are shared via this volume, but I don't want them to overwrite the static files created in the nginx build process (which are sent to `/code/static` in the nginx container. So the volume is mounted in the django container at the path `/code/static` where django collects all the static files, and to `/code/static/django` in the nginx container where it won't overwrite the static files compiled during the nginx build.

Reply:
So I don't fully understand all the steps of reasoning here, but I'm satisfied that this has been thought through and is not an oversight.